src/ast/BindingStore.h

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Maybe we should have `ast/utility/` for files/classes like this?

Reply:
I agree! We should add `ast/utility/` and throw in AstUtils and BindingStore into there. Do you want me to do it as part of this PR?

Reply:
I think separate PR so we can think about it a bit more. There are other files that could also move.